r-package/paper2020/paper2020.Rmd

```diff
@@ -97,10 +108,10 @@ departure_datetime <- as.POSIXct("13-05-2019 14:00:00",
                                  format = "%d-%m-%Y %H:%M:%S")
 
 time_window <- 120 # in minutes
-percentiles <- c(20, 40, 60, 80, 99)
+percentiles <- c(20, 40, 50, 60, 80, 99)
```

Comment:
@mattwigway , I'm not sure we can add more thresholds here. I believe the R5 algorithm now imposes a limit of up to 5 thresolds.  Perhaps @mvpsaraiva can confirm this.

Reply:
Huh, it seemed to work and give reasonable numbers for all percentiles.

Reply:
AnalysisWorkerTask now has a limit of five percentiles, you can see it on [line 46](https://github.com/conveyal/r5/blob/14e59d9645e9999a3b4e6b7684c6de93934564da/src/main/java/com/conveyal/r5/analyst/cluster/AnalysisWorkerTask.java#L46). The `validatePercentiles()` function ([line 245](https://github.com/conveyal/r5/blob/14e59d9645e9999a3b4e6b7684c6de93934564da/src/main/java/com/conveyal/r5/analyst/cluster/AnalysisWorkerTask.java#L245)) enforces that limit. It's a very recent change, from December 2020.

Reply:
Ok. I will do that. Thanks